app/models/vote.rb

```diff
@@ -0,0 +1,6 @@
+class Vote < ApplicationRecord
+  validates :vote, { presence: true }
```

Comment:
Let's name this attribute something other than vote, since `Vote#vote` is redundant.